docs/sql-ref-syntax-aux-cache-uncache-table.md

```diff
@@ -21,11 +21,14 @@ license: |
 
 ### Description
 `UNCACHE TABLE` removes the entries and associated data from the in-memory and/or on-disk cache for a given table or view. The
-underlying entries should already have been brought to cache by previous `CACHE TABLE` operation. `UNCACHE TABLE` on a non-existent table throws Exception if `IF EXISTS` is not specified.
+underlying entries should already have been brought to cache by previous `CACHE TABLE` operation. `UNCACHE TABLE` does not remove 
+a temporary view. `UNCACHE TABLE` on a non-existent table throws an exception if `IF EXISTS` is not specified.
```

Comment:
Do we need to highlight `UNCACHE TABLE does not remove a temporary view.`? It's intuitive that UNCACHE won't drop temp view.

Reply:
Yeah. Make sense. I will update the PR.

Reply:
Done. Please review @cloud-fan